modules/apps/headless/headless-batch-engine/headless-batch-engine-impl/src/main/java/com/liferay/headless/batch/engine/internal/resource/v1_0/ImportTaskResourceImpl.java

```diff
@@ -343,15 +349,21 @@ private Response _getImportTaskContent(
 				batchEngineImportTask.getBatchEngineImportTaskId()),
 			outputStream);
 
+		String normalizedFileName =
+			(fileName != null) ? _normalizeFileName(fileName) :
```

Comment:
ternary operator is not used in Liferay :(
please workout it with `if` statements